package.json

```diff
@@ -10,6 +10,7 @@
     "@okta/okta-signin-widget": "^4.5.1",
     "antd": "^4.8.2",
     "axios": "^0.19.2",
+    "commitizen": "^4.2.2",
```

Comment:
I'm not sure if we want commitizen to be a required dependency. Maybe a devDependency if it is something the entire team will use as a dev tool.

Reply:
I appreciate the reminder about devDependencies. It's been a long time since I've used them, but I will look into them remember how to make comitizen exclusive to developer instances of the codebase.

Reply:
sounds good thaks trevor. after running npm uninstall committizen you can run npm install -D committizen to install as a devDependency. https://bundlephobia.com/result?p=commitizen@4.2.2